components/collapse/collapse.js

```diff
@@ -0,0 +1,129 @@
+import React, {
+  Component,
+  Children,
+  cloneElement,
+  PropTypes,
+} from 'react';
+
+import { getClassNamesWithMods } from '../_helpers';
+import CollapseItem from './collapseItem';
+
+const getNormalizedActiveKey = ({ activeKey = [] }) => {
+  return activeKey instanceof Array ? activeKey : [activeKey];
```

Comment:
I didn't change anything, why I need to create a new array? this function will return array always